pkg/apis/core/validation/validation.go

```diff
@@ -3973,12 +4016,20 @@ func ValidatePodUpdate(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 	// 1.  spec.containers[*].image
 	// 2.  spec.initContainers[*].image
 	// 3.  spec.activeDeadlineSeconds
+	// 4.  spec.terminationGracePeriodSeconds
+	// 5.  spec.containers[*].startupProbe.terminationGracePeriodSeconds
+	// 6.  spec.containers[*].livenessProbe.terminationGracePeriodSeconds
```

Comment:
https://github.com/kubernetes/kubernetes/blob/5ec7b49299ec838ede7f6c064aca95378871f1ec/pkg/apis/core/validation/validation.go#L2793-L2825

No need to veridate initContainers